extension/src/popup/components/PasswordRequirements.tsx

```diff
@@ -1,28 +1,30 @@
 import React from "react";
 import styled from "styled-components";
 
-import { COLOR_PALETTE } from "popup/constants/styles";
-
 const El = styled.div`
   padding: 1.25rem 0 2.5rem;
 `;
 
 const ListEl = styled.ul`
-  color: ${COLOR_PALETTE.secondaryText};
+  color: var(--pal-text-primary);
+  opacity: 0.8;
   list-style-type: disc;
   list-style-position: inside;
-  font-size: 0.9rem;
+  font-size: 1rem;
   padding: 0;
   margin: 0;
+
+  // TODO reconcile with SDS
```

Comment:
the SDS has different bullets than the figma designs, so went with the designs for now. lmk if I should change